tests/Source/SomeParent.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+// class that is used all over the code
+// autoloaded manually via composer to keep the rest of code clean
+namespace SomeNamespace;
+
+abstract class SomeParent
```

Comment:
This class should be located next to the test, here create a new directory `Source`:

https://github.com/rectorphp/rector/tree/master/packages/Php74/tests/Rector/Property/TypedPropertyRector

And make class respect PSR-4 naming, so it's autolaoded by composer by default

Reply:
Done.